models/Discord/Commands/DiscordCommandUserverify.js

```diff
@@ -0,0 +1,32 @@
+const DiscordCommand = require('../DiscordCommand.js');	
+
+class DiscordCommandUserverify extends DiscordCommand {	
+
+    constructor(subsystem) {	
+        super("userverify", "Verifies someone on the discord. Use again to unverify someone.", 'userverify', subsystem);	
```

Comment:
for consistency you might want to use " around then second userverify

Reply:
All the other commands have a ' instead of " around the permission. Would it be affected if it were a  " ?